pages/interface/components/Header/index.js

```diff
@@ -77,6 +78,17 @@ export default function HeaderComponent() {
 
       {user && (
         <>
+          <PrimerHeader.Item
+            sx={{
+              mr: 2,
+            }}>
+            <Tooltip aria-label="Publicar novo conteúdo" direction="s" noDelay={true} wrap={true}>
+              <HeaderLink href="/publicar" aria-label="Publicar novo conteúdo">
+                <MdAdd size={20} color="white" />
```

Comment:
```suggestion
                <PlusIcon size={16} />
```

Motivo: É o ícone oficial da biblioteca de UI do TabNews, então a estilização está mais coerente com o restante do header, por exemplo o alinhamento e o hover.